Terran-One/cw-simulate-ui
# Combine Call History and Debug to Logs

## Issue link

[WL-XXX](https://terran-one.atlassian.net/browse/WL-XXX)

## Description

Combine call history and debug into a logs tab. 

## Test steps

1. Upload WASM contract
2. Execute and verify the logs tab
3.
